### ISSUE TITLE
Make list of keycodes

### ISSUE THREAD

**Issue by Merlin04** [ahtn/keyplus#44]
I'm configuring my keyplus mini firmware, but I don't know what the keycodes are. There should be a list of them.

**Comment by ahtn** [comment]
Yeah, definitely need to work on the documentation, for the mean time you can see a list of key codes here: https://github.com/ahtn/keyplus/blob/master/host-software/keyplus/keycodes/mapped_keycodes.py

If you want to combine a keycode with a modify, you can use a `mod-keycode`, so where mod is one of

* `c` -> control
* `s` -> shift
* `a` -> alt keycode
* `g` -> gui / win key

So to make a key that press `ctrl+v`, use `c-v`.  To make a key that press `ctrl+shift+alt+up` use `csa-up`.
